From 6b8fa5aef9b6895cdab39034dab60b7c12c94b0d Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Thu, 1 Sep 2016 10:42:10 -0700
Subject: [PATCH] MANTA-2863 manta zone setup scripts need to wait for
 config-agent to start Reviewed by: Trent Mick <trentm@gmail.com> Approved by:
 Trent Mick <trentm@gmail.com>

---
 deps/manta-scripts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/manta-scripts b/deps/manta-scripts
index 6c96273..cd72e80 160000
--- a/deps/manta-scripts
+++ b/deps/manta-scripts
@@ -1 +1 @@
-Subproject commit 6c96273f7281df37ba60e4cb8b4cc4dd27ff1f1e
+Subproject commit cd72e801576de5a4b1aec35a7f79409d3f1eebb5
-- 
2.21.0

